src/tests/ftest/scrubber/aggregation.py

```diff
@@ -36,15 +36,24 @@ def test_target_eviction_during_aggregation(self):
         self.pool.set_property("reclaim", "disabled")
         self.add_container(self.pool)
         # Pool and Containers are already created. Just run the IOR.
+        # Run the initial IOR with a small block size.
+        self.ior_cmd.namespace = "/run/ior_small_block_size/*"
+        self.ior_cmd.get_params(self)
         self.run_ior_with_pool(create_cont=False)
         telemetry_string = "engine_pool_vos_aggregation_obj_scanned"
         initial_agg_metrics = self.telemetry.get_metrics(telemetry_string)
-        # Enable the aggregation on the pool.
-        self.pool.set_property("reclaim", "time")
         # Now enable the scrubber on the pool.
         self.pool.set_prop(properties="scrub:timed,scrub_freq:1,scrub_thresh:3")
         initial_metrics = self.scrubber.get_scrub_corrupt_metrics()
+        # The disk fault injection is going to be slow.
+        # Reduce transfer size and increase block size for IOR to run for long time.
+        self.processes = self.params.get("np", "/run/ior_large_block_size/client_processes/*",
+                                         self.processes)
+        self.ior_cmd.namespace = "/run/ior_large_block_size/*"
```

Comment:
We should just use the namespace. It isn't even necessary to use `client_processes` because `*` is recursive
```suggestion
        self.ior_cmd.namespace = "/run/ior_large_block_size/*"
        self.processes = self.params.get("np", self.ior_cmd.namespace, self.processes)
```

Reply:
Done.